bastionzero/target/attributes.go

```diff
@@ -233,3 +234,46 @@ func FlattenControlChannelSummary(ctx context.Context, apiObject *targets.Contro
 		})
 	}
 }
+
+type DatabaseAuthenticationConfig struct {
+	AuthenticationType   types.String `tfsdk:"authentication_type"`
+	CloudServiceProvider types.String `tfsdk:"cloud_service_provider"`
+	Database             types.String `tfsdk:"database"`
+	Label                types.String `tfsdk:"label"`
+}
+
+func DatabaseAuthenticationConfigAttribute() schema.Attribute {
+	return schema.SingleNestedAttribute{
+		Computed:    true,
+		Description: "Information about the db target's database authentication configuration.",
+		Attributes: map[string]schema.Attribute{
+			"authentication_type": schema.StringAttribute{
+				Computed:    true,
+				Description: "The type of authentication used when connecting to the database.",
+			},
+			"cloud_service_provider": schema.StringAttribute{
+				Computed:    true,
+				Description: "Cloud service provider hosting the database. Only used for certain types of authentication, such as ServiceAccountInjection.",
+			},
+			"database": schema.StringAttribute{
+				Computed:    true,
+				Description: "The type of database running on the target.",
+			},
+			"label": schema.StringAttribute{
+				Computed:    true,
+				Description: "User-friendly label for this database authentication configuration.",
+			},
+		},
+	}
+}
+
+func FlattenDatabaseAuthenticationConfig(ctx context.Context, apiObject dbauthconfig.DatabaseAuthenticationConfig) types.Object {
+	attributeTypes, _ := internal.AttributeTypes[ControlChannelSummaryModel](ctx)
```

Comment:
```suggestion
	attributeTypes, _ := internal.AttributeTypes[DatabaseAuthenticationConfigModel](ctx)
```

`ControlChannelSummaryModel` is a different object. Change the generic constraint to `DatabaseAuthenticationConfigModel` (assuming you've also renamed `DatabaseAuthenticationConfig` to `DatabaseAuthenticationConfigModel`), so that this function maps to the correct TF schema model.

Reply:
Copy+paste mistake. Thanks for catching it.